sdk/redisenterprise/arm-redisenterprisecache/test/redisenterprisecache_test.spec.ts

```diff
@@ -49,11 +49,15 @@ describe("RedisEnterprise test", () => {
     await recorder.stop();
   });
 
-  it("operations list test", async () => {
+  it.skip("operations list test", async () => {
```

Comment:
due to our recent policy, we don't block the SDK release if all the other resource types of this new version are available. so skip operation_list and add another list case instead